praw/models/reddit/comment.py

```diff
@@ -228,8 +238,13 @@ def refresh(self):
                 self.id)
 
         # The context limit appears to be 8, but let's ask for more anyway.
+        params = {'context': 100}
+        if 'reply_limit' in self.__dict__:
```

Comment:
How do you feel about `hasattr(self, 'reply_limit')`? And same for `reply_sort` below?

Reply:
Making this change causes BetaMax to throw errors for test_parent__comment and test_parent__chain. I think it is because using hasattr triggers a call to _fetch. I could technically just update the cassettes, but I don't love the idea of changing the timing of the _fetch call when I didn't intend to. Thoughts?

Reply:
Great point. Let's leave it as-in in that case.